core/scripts/umip-2/1_Propose.js

```diff
@@ -0,0 +1,65 @@
+// This script generates and submits UMIP-2 upgrade transactions to the DVM. It can be run on a local ganache
+// fork of the main net or can be run directly on the main net to execute the upgrade transactions.
+// To run this on the localhost first fork main net into Ganache with the proposerWallet unlocked as follows:
+// ganache-cli --fork https://mainnet.infura.io/v3/d70106f59aef456c9e5bfbb0c2cc7164 --unlock 0x2bAaA41d155ad8a4126184950B31F50A1513cE25
+// Then execute the script as: truffle exec ./scripts/umip-2/1_Propose.js --network mainnet-fork from core
+
+const IdentifierWhitelist = artifacts.require("IdentifierWhitelist");
+const Governor = artifacts.require("Governor");
+
+const { RegistryRolesEnum } = require("../../../common/Enums.js");
+
+const tdr = require("truffle-deploy-registry");
+
+const proposerWallet = "0x2bAaA41d155ad8a4126184950B31F50A1513cE25";
+const zeroAddress = "0x0000000000000000000000000000000000000000";
+
+async function runExport() {
+  console.log("Running UMIP-2 Upgrade🔥");
+  console.log("Connected to network id", await web3.eth.net.getId());
+
+  const identifierWhitelist = await IdentifierWhitelist.deployed();
+  const governor = await Governor.deployed();
+
+  // After it's given ownership, the upgrade transaction needs to be executed.
+  const identifierBytes = web3.utils.utf8ToHex("ETH/BTC");
+  const addEthBtcIdentifierTx = identifierWhitelist.contract.methods
+    .addSupportedIdentifier(identifierBytes)
+    .encodeABI();
+
+  console.log("addEthBtcIdentifierTx", addEthBtcIdentifierTx);
+
+  await governor.propose(
```

Comment:
This looks great but do we not want to do the proposal for registering the EMP factory within the same transaction? You could _first_ deploy the emp factory then execute this transaction within the same proposal.

Reply:
The EMP is UMIP-4. I think we want the proposals to be separate since they are separate UMIPs.